src/traccuracy/loaders/_point.py

```diff
@@ -0,0 +1,90 @@
+from typing import Optional, cast
+
+import networkx as nx
+import pandas as pd
+
+from traccuracy._tracking_graph import TrackingGraph
+
+
+def load_point_data(
+    path: str | None = None,
+    df: Optional[pd.DataFrame] = None,  # noqa: UP007, bar syntax breaks docks build
+    parent_column: str = "parent",
+    id_column: str | None = None,
+    pos_columns: tuple[str, ...] = ("z", "y", "x"),
+    time_column: str = "t",
+    name: str | None = None,
+    sep: str | None = None,
+) -> TrackingGraph:
+    """Load point-based tracking data into a TrackingGraph from a csv-like file
+
+    Assumes each row contains:
+    - time
+    - position, e.g. three columns 'z', 'y', 'x'
+    - parent, a reference to the node in the previous time frame
+
+    Args:
+        path (str | None, optional): Path to the csv-like file to load. Defaults to None.
+        df (pd.DataFrame | None, optional): A dataframe that has already been loaded.
+            Defaults to None.
+        parent_column (str | None, optional): A reference to the parent node in the previous
+            time frame. Defaults to "parent".
+        id_column (str | None, optional): Optional column used to specify node ids.
+            Defaults to None and the row index is used as the id
+        pos_columns (tuple[str], optional): A tuple of columns to use for position.
+            Defaults to ("z", "y", "x").
+        time_column (str, optional): The column to use for time. Defaults to "t".
+        name (str | None, optional): Optional string to name/describe the dataset. Defaults to None.
+        sep (str | None, optional): Passed to pd.read_csv to set the sep kwarg. Defaults to None.
+
```

Comment:
Are we envisioning this only used when segmentations aren't present, or additionally when you have a segmentation and have already done your regionprops-like thing to create the graph? I think there is a use case for saving the graph as a CSV alongside the segmentation to avoid the need to re-compute the graph every time you do evaluation. In which case we should have a seg_id column (or whatever the segmentation loaders use to match segs to nodes)